src/firefly/js/ui/SearchPanel.jsx

```diff
@@ -12,12 +12,12 @@ import {useStoreConnector} from './SimpleComponent.jsx';
 import {StatefulTabs, Tab} from './panel/TabPanel.jsx';
 import {makeSearchOnce} from '../util/WebUtil';
 
-const changeSearchOptionOnce= makeSearchOnce(); // setup options to immediately execute the search the first time
+export const changeSearchOptionOnce= makeSearchOnce(); // setup options to immediately execute the search the first time
 
 export function SearchPanel({style={}, initArgs={}}) {
     const searchPanelState = useStoreConnector(getSearchInfo);
 
-    const {activeSearch, groups, allSearchItems={}, flow='vertical', title} = searchPanelState;
+    const {activeSearch, groups, allSearchItems={}, flow='vertical', title, searchStyle} = searchPanelState;
```

Comment:
Adding style into `searchInfo` seem awkward to me.  There has to be a better way.  There's already a `style` prop that you can use to effect the style of this component.  (Btw, you removed it.  This is normally not a good idea.  It may affect existing code.)
There is also a `SearchPanel` class that you can use at the css level.  Regardless, I don't think adding `searchStyle` as a state is a good idea.